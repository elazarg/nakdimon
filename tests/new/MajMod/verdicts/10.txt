יוזכר כִּי הערעורים הנדונים נוֹגְעִים לחיובים בכפל מַס עַל פִּי סְעִיף 50(א) לְחוֹק מַס עֵרֶךְ מוּסָף לְגַבֵּי שְׁנֵי הַיְּחִידִים שבירו ושפיר, וְכֵן שומות תְּשׂוּמוֹת וקנסות עַל אִי נִיהוּל סְפָרִים לְגַבֵּי שְׁנֵי הַתַּאֲגִידִים. בְּעִקְבוֹת הכרעת הַדִּין שֶׁנִּיתְּנָה עַל יְדֵי בֵּית מִשְׁפָּט הַשָּׁלוֹם בְּתֵל אביב-יפו בת"פ 13-03-7455 (וְכֵן דְּחִיַּית הערעורים עַל הכרעת הַדִּין בהמלצת בֵּית הַמִּשְׁפָּט הַמְּחוֹזִי בתל-אביב-יפו בע"פ 15-09-23079) בִּיקֵּשׁ המשיב כִּי הַמִּמְצָאִים והמסקנות בִּפְסַק הַדִּין הַפְּלִילִי יִהְיוּ קבילים כראיות לִכְאוֹרָה לאמור בָּהֶם בְּמִסְגֶּרֶת הערעורים דְּנַן, וְזֹאת מִכּוֹחַ סְעִיף 42א(א) לִפְקוּדַּת הָרְאָיוֹת [נוּסַּח חָדָשׁ], התשל"א-1971. כְּמוֹ כֵּן בִּיקֵּשׁ המשיב כִּי "הערעורים.... ידחו בְּסוּגְיוֹת שהוכרעו בַּהֲלִיךְ הַפְּלִילִי" (סְעִיף 19(בְּ) לַבַּקָּשָׁה). יֵשׁ לְצַיֵּין כִּי המשיב עתר לִפְסַק דִּין "חֶלְקֵי" בִּלְבַד כִּי הֲרֵי לֹא קַיֶּימֶת חֲפִיפָה מְלֵאָה בֵּין חֶשְׁבּוֹנִיּוֹת הַמַּס שנדונו בַּהֲלִיךְ הַפְּלִילִי לְבֵין אֵלּוּ שֶׁהֵן מוּשָּׂא ערעורי הַמַּס הָאֶזְרָחִיִּים. לָעִנְיָין אַחֲרוֹן זֶה אשוב בַּהֶמְשֵׁךְ. אֶתְיַיחֵס רֵאשִׁית לחיובים שֶׁהוּטְּלוּ עַל הַיְּחִידִים לְפִי סְעִיף 50(א) לְחוֹק בְּקֶשֶׁר לְהוֹצָאַת חֶשְׁבּוֹנִיּוֹת שֶׁלֹּא כַּדִּין. הָאִישּׁוּם הַפְּלִילִי בְּנוֹשֵׂא זֶה (לְאַחַר צמצומו) הִתְיַיחֵס ל-304 חֶשְׁבּוֹנִיּוֹת מַס אֲשֶׁר נִמְסְרוּ לִידֵי שְׁלוֹשָׁה עֶשֶׂר גּוֹרְמִים שׁוֹנִים (ה"המקזזים" כלשון הכרעת הַדִּין), כַּאֲשֶׁר סְכוּם המע"מ הַכּוֹלֵל הַגָּלוּם בָּהֶן מַגִּיעַ לִכְדֵי 8.13 מִילְיוֹן ש"ח.