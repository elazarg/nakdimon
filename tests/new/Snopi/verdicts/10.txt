יוּזְכַּר כִּי הערעורים הַנִּדּוֹנִים נוֹגְעִים לחיובים בְּכֶפֶל מַס עַל פִּי סְעִיף 50(א) לְחוֹק מַס עָרַךְ מוּסָף לְגַבִּי שְׁנֵי הַיְּחִידִים שבירו וְשֶׁפִּיר, וְכֵן שׁוּמוֹת תשומות וּקְנָסוֹת עַל אִי נִיהוּל סַפָּרִים לְגַבֵּי שְׁנֵי הַתַּאֲגִידִים.
בַּעֲקֵבוֹת הַכְרָעַת הַדִּין שֶׁנִּיתְּנָה עַל יְדֵי בֵּית מִשְׁפָּט הַשָּׁלוֹם בְּתֵל אָבִיב-יִפּוּ בת"פ 13-03-7455 (וְכֵן דחיית הערעורים עַל הַכְרָעַת הַדִּין בְּהַמְלָצַת בֵּית הַמִּשְׁפָּט הַמְּחוֹזִי בְּתֵל-אָבִיב-יִפּוּ בע"פ 15-09-23079) בִּיקֵּשׁ הַמֵּשִׁיב כִּי הַמִּמְצָאִים וְהַמַּסְקָנוֹת בִּפְסַק הַדִּין הַפְּלִילִי יִהְיוּ קְבִילִים כִּרְאָיוֹת לִכְאוֹרָה לָאָמוּר בָּהֶם בַּמִּסְגֶּרֶת הערעורים דנן, וְזֹאת מִכּוֹחַ סְעִיף 42א(א) לִפְקוּדַּת הָרְאָיוֹת [נוֹסַח חָדָשׁ], התשל"א-1971.
כְּמוֹ כֵן בִּיקֵּשׁ הַמֵּשִׁיב כִּי "הערעורים.... יִדְחוּ בַּסּוּגְיוֹת שהוכרעו בַּהֲלִיךְ הַפְּלִילִי" (סְעִיף 19(ב) לַבַּקָּשָׁה).
יֵשׁ לְצַיֵּין כִּי הַמֵּשִׁיב עָתַר לִפְסַק דִּין "חֶלְקִי" בִּלְבַד כִּי הֲרֵי לֹא קִיַּימְתָּ חֲפִיפָה מְלֵאָה בֵּין חשבוניות הַמַּס שֶׁנִּדּוֹנּוּ בַּהֲלִיךְ הַפְּלִילִי לְבֵין אֵלּוּ שֶׁהֵן מושא עִרְעוּרִי הַמַּס הָאֶזְרָחִיִּים.
לְעִנְיָין אַחֲרוֹן זֶה אָשׁוּב בַּהֶמְשֵׁךְ.
אֶתְיַיחֵס רָאשִׁית לחיובים שֶׁהוּטְּלוּ עַל הַיְּחִידִים לְפִי סְעִיף 50(א) לְחוֹק בַּקֶּשֶׁר לְהוֹצָאַת חשבוניות שֶׁלֹּא כְּדִין.
הָאִישּׁוּם הַפְּלִילִי בְּנוֹשֵׂא זֶה (לְאַחֵר צִמְצוּמוֹ) הִתְיַיחֵס ל-304 חשבוניות מַס אֲשֶׁר נִמְסְרוּ לְיָדִי שְׁלוֹשָׁה עָשָׂר גּוֹרְמִים שׁוֹנִים (ה"המקזזים" כִּלְשׁוֹן הַכְרָעַת הַדִּין), כַּאֲשֶׁר סְכוּם הַמָּעָ"מ הַכּוֹלֵל הַגָּלוּם בָּהֶן מַגִּיעַ לִכְדִי 8.13 מִילְיוֹן שָׂ"ח.