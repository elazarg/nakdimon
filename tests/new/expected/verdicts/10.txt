יוּזְכַּר כִּי הָעִרְעוּרִים הַנְּדוֹנִים נוֹגְעִים לְחִיּוּבִים בְּכֶפֶל מַס עַל פִּי סְעִיף 50(א) לְחוֹק מַס עֵרֶךְ מוּסָף לְגַבֵּי שְׁנֵי הַיְּחִידִים שְׁבִּירוֹ וְשָׁפִיר, וְכֵן שׁוּמוֹת תְּשׂוּמוֹת וּקְנָסוֹת עַל אִי נִיהוּל סְפָרִים לְגַבֵּי שְׁנֵי הַתַּאֲגִידִים.
בְּעִקְבוֹת הַכְרָעַת הַדִּין שֶׁנִּיתְּנָה עַל יְדֵי בֵּית מִשְׁפַּט הַשָּׁלוֹם בְּתֵל אָבִיב-יָפוֹ בת"פ 13-03-7455 (וְכֵן דְּחִיַּית הָעִרְעוּרִים עַל הַכְרָעַת הַדִּין בְּהַמְלָצַת בֵּית הַמִּשְׁפָּט הַמְּחוֹזִי בְּתֵל-אָבִיב-יָפוֹ בע"פ 15-09-23079) בִּיקֵּשׁ הַמֵּשִׁיב כִּי הַמִּמְצָאִים וְהַמַּסְקָנוֹת בִּפְסַק הַדִּין הַפְּלִילִי יִהְיוּ קְבִילִים כִּרְאָיוֹת לִכְאוֹרָה לָאָמוּר בָּהֶם בְּמִסְגֶּרֶת הָעִרְעוּרִים דְּנַן, וְזֹאת מִכּוֹחַ סְעִיף 42א(א) לִפְקוּדַּת הָרְאָיוֹת [נוֹסַח חָדָשׁ], התשל"א-1971.
כְּמוֹ כֵן בִּיקֵּשׁ הַמֵּשִׁיב כִּי "הָעִרְעוּרִים.... יִדְחוּ בַּסּוּגְיוֹת שֶׁהוּכְרְעוּ בַּהֲלִיךְ הַפְּלִילִי" (סְעִיף 19(ב) לַבַּקָּשָׁה).
יֵשׁ לְצַיֵּין כִּי הַמֵּשִׁיב עָתַר לִפְסַק דִּין "חֶלְקִי" בִּלְבַד כִּי הֲרֵי לֹא קַיֶּימֶת חֲפִיפָה מְלֵאָה בֵּין חֶשְׁבּוֹנִיּוֹת הַמַּס שֶׁנָּדוֹנוּ בַּהֲלִיךְ הַפְּלִילִי לְבֵין אֵלּוּ שֶׁהֵן מוּשַּׂא עִרְעוּרֵי הַמַּס הָאֶזְרָחִיִּים.
לְעִנְיָין אַחֲרוֹן זֶה אָשׁוּב בַּהֶמְשֵׁךְ.
אֶתְיַיחֵס רֵאשִׁית לַחִיּוּבִים שֶׁהוּטְּלוּ עַל הַיְּחִידִים לְפִי סְעִיף 50(א) לַחוֹק בְּקֶשֶׁר לְהוֹצָאַת חֶשְׁבּוֹנִיּוֹת שֶׁלֹּא כַּדִּין.
הָאִישּׁוּם הַפְּלִילִי בְּנוֹשֵׂא זֶה (לְאַחַר צִמְצוּמוֹ) הִתְיַיחֵס לְ-304 חֶשְׁבּוֹנִיּוֹת מַס אֲשֶׁר נִמְסְרוּ לִידֵי שְׁלוֹשָׁה עָשָׂר גּוֹרְמִים שׁוֹנִים (הַ"הַמְּקַזְּזִים" כִּלְשׁוֹן הַכְרָעַת הַדִּין), כַּאֲשֶׁר סְכוּם הַמַּעַ"מ הַכּוֹלֵל הַגָּלוּם בָּהֶן מַגִּיעַ לִכְדֵי 8.13 מִילְיוֹן שַׁ"ח.
